.github/workflows/main.yml

```diff
@@ -2,10 +2,15 @@
 on:
   pull_request:
     types: [opened, edited, reopened, synchronize]
+    branches: 
+      - '*'
+      - '*/*'
+      - '**'    
+      - '!main'
+      - '!master'
 
 jobs:
   do_it:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
```

Comment:
We need to keep in mind that this will cause problems in workflows like [core's PR linter](https://github.com/warp-ds/core/blob/alpha/.github/workflows/pr-title-linter.yml#L6-L11) where we removed `actions/checkout@v3` step.

Reply:
Ok do we need to have this step in order to change the PR title?

Reply:
I thought we did 🤔 Maybe we could keep this bit here for now so we can test by copy-pasting this workflow's code in the release-playground. Wdyt?